plugin.yaml

```diff
@@ -132,6 +140,30 @@ node_types:
         poststart:
           implementation: starlingx.cloudify_starlingx.resources.wrcp.poststart
 
+  cloudify.nodes.starlingx.PATCHES_LIST:
+    derived_from: cloudify.nodes.Root
+    properties:
+      <<: *external_resource
+      <<: *client_config
+      resource_config:
+        type:  cloudify.types.starlingx.ClientConfig
+    interfaces:
+      cloudify.interfaces.lifecycle:
+        start:
+          implementation: starlingx.cloudify_starlingx.resources.client_api.get_patches_list
+
+  cloudify.nodes.starlingx.PATCHE_DETAILS:
+    derived_from: cloudify.nodes.Root
```

Comment:
We should use camel case on the final element in a node type or relationship type. The reason that `cloudify.nodes.starlingx.WRCP` is not camel case is because it is an abbreviation. But `PATCHE_DETAILS` can be `PatchDetails`.

Reply:
Fixed